third_party/ruby/build-ruby.bzl

```diff
@@ -151,11 +151,6 @@ cp "$out_dir/bin/bundle" "$out_dir/bin/bundler"
 
 {install_gems}
 
-# Since we get our version of bundler from update_rubygems, we have to apply
-# this patch after everything is built. This is a bit of a hack, but the need
-# for the patch should go away once we upgrade Ruby and/or bundler anyway.
-{post_build_patch_command}
```

Comment:
Should we also delete where `post_build_patch_command`/`post_build_patch_commands` is defined?

Reply:
I think that's a good idea, but I wanted to do that on a subsequent PR to minimize the diff.